src/sentry/static/sentry/app/components/group/releaseChart.jsx

```diff
@@ -21,15 +18,16 @@ const GroupReleaseChart = createReactClass({
     firstSeen: PropTypes.string,
     lastSeen: PropTypes.string,
     title: PropTypes.string,
-  },
+  };
 
-  getInitialState() {
-    return this.getNextState(this.props);
-  },
+  constructor(...args) {
+    super(...args);
+    this.state = this.getNextState(args[0]);
```

Comment:
Should we define the param name as props?

alternatively, class props should work too

`
state = this.getNextState(this.props);
`